src/client/components/RoomList/RoomList.js

```diff
@@ -8,9 +8,9 @@ const RoomList = ({ roomList }) => {
     <div className="outside">
       {roomList.map((room) => {
         return (
-          <div>
+          <div key={room.title}>
```

Comment:
Do you really need to have additional `div` here for styling? You should have `ul` before starting to map and you should return `li` directly instead of it being in a `div`.

Reply:
Also, you should really strive towards using `room.id` here - do you have API for this component? I am pretty sure you should be able to get an `id` here and also add it in line 28.

Reply:
i removed divs and changed it to ul and li